src/bootstrap/dist.rs

```diff
@@ -1897,6 +1899,26 @@ fn add_env(builder: &Builder<'_>, cmd: &mut Command, target: TargetSelection) {
     }
 }
 
+fn install_llvm_file(builder: &Builder<'_>, source: &Path, destination: &Path) {
+    if source.as_os_str().is_empty() {
```

Comment:
This looks iffy - what hits this path? Maybe we can avoid calling the function instead?